controllers/subscription_controller.go

```diff
@@ -67,15 +68,29 @@ type SubscriptionReconciler struct {
 //+kubebuilder:rbac:groups=operators.coreos.com,resources=clusterserviceversions/finalizers,verbs=update
 //+kubebuilder:rbac:groups=operators.coreos.com,resources=installplans,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=operators.coreos.com,resources=operatorconditions,verbs=get;list;watch
+//+kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch
 //+kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=mutatingwebhookconfigurations,verbs=get;list;watch;create;update
 
 func (r *SubscriptionReconciler) Reconcile(ctx context.Context, _ ctrl.Request) (ctrl.Result, error) {
-
 	logger := log.FromContext(ctx)
+	logger.Info("starting reconcile")
 
-	olmPkgRecords := getOlmPkgRecord()
+	configmap, err := GetOdfConfigMap(ctx, r.Client, logger)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
 
-	if err := r.setOperatorCondition(logger, olmPkgRecords); err != nil {
+	var olmPkgRecords []*OlmPkgRecord
+	var csvNames []string
+	ParseOdfConfigMapRecords(logger, configmap, func(record *OlmPkgRecord) {
+		if record.Pkg != "" && record.Csv != "" && record.Channel != "" {
+			csvNames = append(csvNames, record.Csv)
+			olmPkgRecords = append(olmPkgRecords, record)
+		}
+	})
+	logger.Info("subscriptions records", "olmPkgRecords", olmPkgRecords)
```

Comment:
I would say this is more of a debug info then an "info" into. Can we log this using debug level?

Reply:
We don't have Debug. We can evaluate this as a tech debt.